### PR TITLE
Prevent duplication story during csv import

### PR DESCRIPTION
**Pull request by mfcruz**
Solution for #224 
Uniqueness is based on story title.
Adding model validation will also prevent further duplication of entry.

### CODE REVIEW

File: app/models/story.rb [hunk 1]
```diff
@@ -8,32 +8,32 @@ class Story < ApplicationRecord
...
+        # Add interviewer
+        story.interviewer = row[6].blank? ? nil : Speaker.find_or_create_by(name: row[6])
+        # Add Language
+        story.language = row[7].blank? ? nil : row[7]
+        if row[8] && File.exist?(Rails.root.join('media', row[8]))
+          file = File.open(Rails.root.join('media',row[8]))
+          story.media.attach(io: file, filename: row[8])
+        end
+        story.permission_level = row[9].blank? ? "anonymous" : "user_only"
       end
```
Comment (mirandawang) [inline app/models/story.rb]: Missing a `)` parenthesis here :)

**Comment by rudokemper** [comment]
I do think it would be good to distinguish stories on a more substantial basis than just story title alone.

To illustrate, imagine that there are two stories titled "Oral History of St. Petersburg". One is about St. Petersburg, Florida and the other about St. Petersburg, Russia. Distinct stories and places, but same title.

In all likelihood, Terrastories won't be used to map oral histories about Central Florida and Russia in the same instance, but in indigenous territories there are distinct places with the same name too.

Or imagine a case where Terrastories is being used to map family histories within a community. And multiple people want to use the title "My Grandma's History" or something like that. I don't think we should expect users to ensure their stories have unique titles.

But then, I'm not sure what could guarantee uniqueness for Stories? My thinking was video filename, but what if there are multiple `MyGrandmasHistory.mp4`s? Same issue.

What if it just checks for _all_ of the fields in the story model? I think that would be robust enough for sure. And we can just make a note somewhere in the documentation for users to avoid across-the-board duplication of field content in the spreadsheets.

Thoughts?

**Comment by midhunkrishna** [comment]
@rudokemper may be title, locations and content?

**Comment by rudokemper** [comment]
Sounds good!

**Comment by mirandawang** [comment]
Hey @mfcruz , are you still able to work on this? :)

**Comment by mirandawang** [comment]
Closing due to inactivity

**Pull request closed without merging**